README.md

```diff
@@ -209,6 +214,89 @@ If you set `passReqToCallback` option to true, you can use one of the following
   function(req, profile, done)
 ```
 
+#### 5.1.1.4 JWE support
+
+  We support encrypted id_token in JWE Compact Serialization format. The key encryption algorithms supported are: 
+  `RSA1_5`, `RSA-OAEP`, `A128KW`, `A256KW`, `dir`; the content encryption algorithms supported are:
+  `A128CBC-HS256`, `A192CBC-HS384`, `A256CBC-HS512`, `A128GCM`, and `A256GCM`.
+
+  In order to encrypt the id_token, keys have to be provided in JWK format using `jweKeyStore` option. We will first
+  try the key with the corresponding kid. If decryption fails, we will try every possible key in `jweKeyStore`. 
+  The following is an example of `jweKeyStore`:
+
+  ```javascript
+
+    jweKeyStore: [ 
+      { 'kid': 'sym_key_256', 'kty': 'oct', 'k': 'WIVds2iwJPwNhgUgwZXmn/46Ql1EkiL+M+QqDRdQURE=' }, 
+      { 'kid': 'sym_key_128', 'kty': 'oct', 'k': 'GawgguFyGrWKav7AX4VKUg'}, 
+      { 'kid': 'sym_key_384', 'kty': 'oct', 'k': 'AAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8gISIjJCUmJygpKissLS4v'},
+      { 'kid': 'rsa_key', 
```

Comment:
Do we provide 'encryption' services? If not, then just the public key is needed.